xsd/siri_model/siri_situationActions.xsd

```diff
@@ -499,7 +499,7 @@ Rail transport, Roads and road transport
     <xsd:sequence>
      <xsd:element name="WorkgroupRef" type="xsd:string" default="true" minOccurs="0">
```

Comment:
"true" is a strange default for a workgroup! No default makes more sense.

Reply:
done, also changed in UserRef
![image](https://user-images.githubusercontent.com/1889203/122420124-b80a0400-cf8b-11eb-91eb-58a9b8c01e0e.png)

Reply:
:-)